frontend/server/src/Controllers/Problem.php

```diff
@@ -2719,18 +2739,18 @@ public static function apiClarifications(\OmegaUp\Request $r): array {
             'problem_alias'
         );
         $problem = \OmegaUp\DAO\Problems::getByAlias($r['problem_alias']);
-        if (is_null($problem)) {
+        if (is_null($problem) || is_null($problem->problem_id)) {
             throw new \OmegaUp\Exceptions\NotFoundException('problemNotFound');
         }
 
-        $is_problem_admin = \OmegaUp\Authorization::isProblemAdmin(
+        $problemAdmin = \OmegaUp\Authorization::isProblemAdmin(
```

Comment:
nit:

```suggestion
        $isProblemAdmin = \OmegaUp\Authorization::isProblemAdmin(
```

si no, se puede confundir con el hecho de que esa variable contiene un usuario o una identidad que es el administrador.

Reply:
Listo